sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/util/StringUtil.java

```diff
@@ -67,4 +67,14 @@ public static boolean isLongValue(final String value) {
             return false;
         }
     }
+    
+    /**
+     * Judge string is empty or not.
+     *
+     * @param value to be judged string value
+     * @return is Empty string or not
+     */
+    public static boolean isEmpty(final String value) {
```

Comment:
Should use guava's `Strings.isNullOrEmtpy()` to instead this method, please remove this method.